flake.nix

```
@@ -103,6 +103,14 @@
             self.nixosModules.mixins-terminfo
           ];
         };
+        example-mixins-trusted-caches = nixpkgs.lib.nixosSystem {
+          system = "x86_64-linux";
+          modules = [
+            dummy
+            fake-hardware
+            self.nixosModules.mixins-trusted-caches
```

```suggestion
            self.nixosModules.mixins-trusted-nix-caches
```
To be a bit more descriptive